docs/feature-store/feature-vectors.md

```diff
@@ -214,8 +256,9 @@ resp = fs.get_offline_features(
     with_indexes=False,
 )
 ```
+    
 
-### Using an online feature vector
+## Using an online feature vector
```

Comment:
@assaf758 @jillnogold The offline gets a fully description while the online is very basic, maybe we need to add more description to this part like we have in the offline section?